src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -286,11 +286,16 @@ namespace winrt::TerminalApp::implementation
             // that the window size is _first_ set up as something sensible, so
             // leaving fullscreen returns to a reasonable size.
             const auto launchMode = this->GetLaunchMode();
-            if (launchMode == LaunchMode::FullscreenMode)
+            if (IsQuakeWindow())
+            {
+                _root->ToggleFocusMode();

```

Comment:
Why `ToggleFocusMode` and not something like `EnableFocusMode` or `SetFocusMode(true)`? QM will always be in focus mode right?

Reply:
Frankly, I hadn't written `_SetFocusMode` yet when I added this line 😋

Reply:
Ah, but that's a private member on TerminalPage... meh, I just didn't feel like adding another method that'll only have one usage right here